examples/esp32c6/Cargo.toml

```diff
@@ -6,13 +6,14 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-esp32c6-hal = { version = "0.8.0", features = ["eh1", "async", "embassy", "embassy-time-systick", "embassy-executor-thread"] }
+esp-hal = { version = "0.16.1", features = ["esp32c6", "eh1", "async", "embassy", "embassy-time-systick-16mhz", "embassy-executor-thread"] }
```

Comment:
Could you also add the feature `embassy-integrated-timers` to esp-hal?

Reply:
Done